app/Services/MoviesService.php

```diff
@@ -63,4 +63,14 @@ public function create(array $movie, User $user): Movie
         $movie['user_id'] = $user->id;
         return Movie::create($movie);
     }
+
+    /**
+     * Find top 10 most liked movies for Movies table
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function getTopTenMovies()
+    {
+        return Movie::all()->SortByDesc('total_likes')->values()->take(10);
```

Comment:
Ne, ne moj sortirati u php-u, order u bazi radi i LIMIT 10. 
```suggestion
        return Movie::orderBy('total_likes', 'DESC')->take(10)->get();
```
Tako nesto kontam